ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/_client.py

```diff
@@ -72,28 +73,42 @@ def callback(response_future):
                 return
             response = response_future.result()
             rpc_info.response = response
+            if "ByteSize" in dir(response):
+                metrics_recorder.record_bytes_in(
+                    response.ByteSize(), client_info.full_method
+                )
 
     return callback
 
 
 class OpenTelemetryClientInterceptor(
     grpcext.UnaryClientInterceptor, grpcext.StreamClientInterceptor
 ):
-    def __init__(self, tracer):
+    def __init__(self, tracer, exporter, interval):
         self._tracer = tracer
+        self._meter = metrics.get_meter(__name__)
```

Comment:
If the user is not using the default SDK, `DefaultMeter` will be used (or some other Meter implementation) which might not have the `collect` method defined, which will cause this to crash.

Reply:
Not sure what you mean here? I took this part from ext/system_metrics, is that wrong too?

Reply:
I'm not sure about ext/system_metrics, but if you are not explicitly using the meter implementation in `opentelemetry-sdk`, then this would crash, since every `tick()` in `PushController` calls meter.collect() which only exists in the sdk implementation.

Reply:
Also on that note, `opentelemetry-sdk` would need to be a dependency now correct in `setup,cfg` correct?

Reply:
So I should create the meterprovider from opentelemetry-sdk explicitly here even though that means the user can't set it?

Reply:
I think you can simply just take a dependency on the sdk and include in the examples that you must set meterprovider. Also, since users can autoinstrument the grpc instrumentor, you must not have this logic execute (since meterprovider won't be set).